apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -79,7 +79,9 @@ impl BridgeQueryPlanner {
 
                     if has_validation_errors && !schema.has_errors() {
                         tracing::warn!(
-                            monotonic_counter.apollo_router_schema_validation_false_negative = 1,
+                            monotonic_counter.apollo.router.validation = 1,
+                            validation.source = "schema",
+                            validation.result = "false_negative",
```

Comment:
I've approved the PR, but it might be a "good thing" to use `const` or `enum` or something to replace strings like "schema", "false_negative", etc... where repetition may introduce typos. There aren't any typos, but I'm thinking about future changes and the risk of diversion.

Reply:
Yeah. I'll use consts as the most lightweightest option since this should be temporary code anyways.